ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -1699,3 +1699,34 @@ FCU_DSP_DFVSC_DTM_RTM:
   - DTM
   - RTM
 
+
+FCU_DXZTC_DFSS:
+  guid: "5688519e-f88e-4620-8bd5-89d91467028d"
+  description: "FCU with SS, discharge temp sensors, zone control and sensor, filter change"
+  is_canonical: true
+  implements:
+  - FCU
+  - DXZTC
+  - DFSS
+  opt_uses:
+  - failed_alarm
+  - filter_alarm
+
+FCU_RHDHS_DXDC_DFSS_RTM:
+  guid: "f60a2563-da8f-4360-b2d9-f471c065356c"
+  description: "FCU with temperature & pressure monitor, humidity monitor and water leak & status alarm."
+  is_canonical: true
+  implements:
+  - FCU
+  - RHDHS
+  - DXDC
+  - DFSS
+  - RTM
+  uses:
+  - run_status
```

Comment:
How is this `run_status` different from `discharge_fan_run_status` (that is included in the `DFSS` type?

Reply:
run_status checks if the whole unit is on/off. as I didn't see this point anywhere, should it be in opt_uses section?

Reply:
This is unnecessary to include as we typically can infer if the unit is running based on the `discharge_fan_run_status` or the status of the compressors. Please remove this field